msagastya/task1.html

```diff
@@ -0,0 +1,18 @@
+<html>
+    <head>
+        <title>task1</title>
+    </head>
+    <body>
+        <h1 style="color: red;">First Task</h1>
+        <p style="max-width: 400px; text-align: justify;">Hare Ram Hare Ram
+            Ram Ram Hare Hare
+            Hare Krishna Hare Krishna
+            Krishna Krishna Hare Hare
+        </p>
+        <ul>
+            <li>Suyash Mishra</li>
+        </ul>
+        Github Link :
+        <a href="https://github.com/msagastya">Suyash Mishra</a>
```

Comment:
The task says that you should create a link in the list over your name, but you have created another link.
```suggestion
        <ul>
            <li><a href="https://github.com/msagastya">Suyash Mishra</a></li>
        </ul>
```